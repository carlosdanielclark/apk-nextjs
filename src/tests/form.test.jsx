import { mount, shallow } from 'enzyme';
import Form from '../app/form';

describe('Tests in "Form"', () => {

    test('Must show correctly', () => {
        //(1) initialization
        const wrapper = shallow( <Form /> );
        //(2) Stimulus  and  //(3) Observation 
        expect( wrapper ).toMatchSnapshot();
    });
  
   
})