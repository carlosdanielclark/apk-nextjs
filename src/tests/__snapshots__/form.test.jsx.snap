// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tests in "Form" > Must show correctly 1`] = `
<div>
  <h3
    className="c-red mb-2"
  >
     Drag the arrow to dropdown buttons and insert amount
  </h3>
  <div
    className="_Container_aa56d9"
  >
    <form
      className="_Form_aa56d9"
      onSubmit={[Function]}
    >
      <div
        className="_Form_Container_aa56d9"
      >
        <h2>
          From
        </h2>
        <div
          className="_Content_aa56d9"
        >
          <div
            class="_dropdown_aa56d9"
          >
            <span>
               
              Select Base
               
            </span>
            <div
              class="_dropdown_content_aa56d9"
            >
              <p
                onClick={[Function]}
              >
                EUR
              </p>
            </div>
          </div>
          <input
            className="_Input_aa56d9"
            id="input_base"
            name="base"
            onChange={[Function]}
            placeholder="Amount"
            type="number"
            value={1}
          />
        </div>
        <h2>
           to 
        </h2>
        <div
          className="_Content_aa56d9"
        >
          <div
            class="_dropdown_aa56d9"
          >
            <span>
               
              Select Rate
               
            </span>
            <div
              class="_dropdown_content_aa56d9"
              id="change_rate"
            >
              <p
                onClick={[Function]}
              >
                AED
              </p>
              <p
                onClick={[Function]}
              >
                AFN
              </p>
              <p
                onClick={[Function]}
              >
                ALL
              </p>
              <p
                onClick={[Function]}
              >
                AMD
              </p>
              <p
                onClick={[Function]}
              >
                ANG
              </p>
              <p
                onClick={[Function]}
              >
                AOA
              </p>
              <p
                onClick={[Function]}
              >
                ARS
              </p>
              <p
                onClick={[Function]}
              >
                AUD
              </p>
              <p
                onClick={[Function]}
              >
                AWG
              </p>
              <p
                onClick={[Function]}
              >
                AZN
              </p>
              <p
                onClick={[Function]}
              >
                BAM
              </p>
              <p
                onClick={[Function]}
              >
                BBD
              </p>
              <p
                onClick={[Function]}
              >
                BDT
              </p>
              <p
                onClick={[Function]}
              >
                BGN
              </p>
              <p
                onClick={[Function]}
              >
                BHD
              </p>
              <p
                onClick={[Function]}
              >
                BIF
              </p>
              <p
                onClick={[Function]}
              >
                BMD
              </p>
              <p
                onClick={[Function]}
              >
                BND
              </p>
              <p
                onClick={[Function]}
              >
                BOB
              </p>
              <p
                onClick={[Function]}
              >
                BRL
              </p>
              <p
                onClick={[Function]}
              >
                BSD
              </p>
              <p
                onClick={[Function]}
              >
                BTC
              </p>
              <p
                onClick={[Function]}
              >
                BTN
              </p>
              <p
                onClick={[Function]}
              >
                BWP
              </p>
              <p
                onClick={[Function]}
              >
                BYN
              </p>
              <p
                onClick={[Function]}
              >
                BYR
              </p>
              <p
                onClick={[Function]}
              >
                BZD
              </p>
              <p
                onClick={[Function]}
              >
                CAD
              </p>
              <p
                onClick={[Function]}
              >
                CDF
              </p>
              <p
                onClick={[Function]}
              >
                CHF
              </p>
              <p
                onClick={[Function]}
              >
                CLF
              </p>
              <p
                onClick={[Function]}
              >
                CLP
              </p>
              <p
                onClick={[Function]}
              >
                CNY
              </p>
              <p
                onClick={[Function]}
              >
                COP
              </p>
              <p
                onClick={[Function]}
              >
                CRC
              </p>
              <p
                onClick={[Function]}
              >
                CUC
              </p>
              <p
                onClick={[Function]}
              >
                CUP
              </p>
              <p
                onClick={[Function]}
              >
                CVE
              </p>
              <p
                onClick={[Function]}
              >
                CZK
              </p>
              <p
                onClick={[Function]}
              >
                DJF
              </p>
              <p
                onClick={[Function]}
              >
                DKK
              </p>
              <p
                onClick={[Function]}
              >
                DOP
              </p>
              <p
                onClick={[Function]}
              >
                DZD
              </p>
              <p
                onClick={[Function]}
              >
                EGP
              </p>
              <p
                onClick={[Function]}
              >
                ERN
              </p>
              <p
                onClick={[Function]}
              >
                ETB
              </p>
              <p
                onClick={[Function]}
              >
                EUR
              </p>
              <p
                onClick={[Function]}
              >
                FJD
              </p>
              <p
                onClick={[Function]}
              >
                FKP
              </p>
              <p
                onClick={[Function]}
              >
                GBP
              </p>
              <p
                onClick={[Function]}
              >
                GEL
              </p>
              <p
                onClick={[Function]}
              >
                GGP
              </p>
              <p
                onClick={[Function]}
              >
                GHS
              </p>
              <p
                onClick={[Function]}
              >
                GIP
              </p>
              <p
                onClick={[Function]}
              >
                GMD
              </p>
              <p
                onClick={[Function]}
              >
                GNF
              </p>
              <p
                onClick={[Function]}
              >
                GTQ
              </p>
              <p
                onClick={[Function]}
              >
                GYD
              </p>
              <p
                onClick={[Function]}
              >
                HKD
              </p>
              <p
                onClick={[Function]}
              >
                HNL
              </p>
              <p
                onClick={[Function]}
              >
                HRK
              </p>
              <p
                onClick={[Function]}
              >
                HTG
              </p>
              <p
                onClick={[Function]}
              >
                HUF
              </p>
              <p
                onClick={[Function]}
              >
                IDR
              </p>
              <p
                onClick={[Function]}
              >
                ILS
              </p>
              <p
                onClick={[Function]}
              >
                IMP
              </p>
              <p
                onClick={[Function]}
              >
                INR
              </p>
              <p
                onClick={[Function]}
              >
                IQD
              </p>
              <p
                onClick={[Function]}
              >
                IRR
              </p>
              <p
                onClick={[Function]}
              >
                ISK
              </p>
              <p
                onClick={[Function]}
              >
                JEP
              </p>
              <p
                onClick={[Function]}
              >
                JMD
              </p>
              <p
                onClick={[Function]}
              >
                JOD
              </p>
              <p
                onClick={[Function]}
              >
                JPY
              </p>
              <p
                onClick={[Function]}
              >
                KES
              </p>
              <p
                onClick={[Function]}
              >
                KGS
              </p>
              <p
                onClick={[Function]}
              >
                KHR
              </p>
              <p
                onClick={[Function]}
              >
                KMF
              </p>
              <p
                onClick={[Function]}
              >
                KPW
              </p>
              <p
                onClick={[Function]}
              >
                KRW
              </p>
              <p
                onClick={[Function]}
              >
                KWD
              </p>
              <p
                onClick={[Function]}
              >
                KYD
              </p>
              <p
                onClick={[Function]}
              >
                KZT
              </p>
              <p
                onClick={[Function]}
              >
                LAK
              </p>
              <p
                onClick={[Function]}
              >
                LBP
              </p>
              <p
                onClick={[Function]}
              >
                LKR
              </p>
              <p
                onClick={[Function]}
              >
                LRD
              </p>
              <p
                onClick={[Function]}
              >
                LSL
              </p>
              <p
                onClick={[Function]}
              >
                LTL
              </p>
              <p
                onClick={[Function]}
              >
                LVL
              </p>
              <p
                onClick={[Function]}
              >
                LYD
              </p>
              <p
                onClick={[Function]}
              >
                MAD
              </p>
              <p
                onClick={[Function]}
              >
                MDL
              </p>
              <p
                onClick={[Function]}
              >
                MGA
              </p>
              <p
                onClick={[Function]}
              >
                MKD
              </p>
              <p
                onClick={[Function]}
              >
                MMK
              </p>
              <p
                onClick={[Function]}
              >
                MNT
              </p>
              <p
                onClick={[Function]}
              >
                MOP
              </p>
              <p
                onClick={[Function]}
              >
                MRO
              </p>
              <p
                onClick={[Function]}
              >
                MUR
              </p>
              <p
                onClick={[Function]}
              >
                MVR
              </p>
              <p
                onClick={[Function]}
              >
                MWK
              </p>
              <p
                onClick={[Function]}
              >
                MXN
              </p>
              <p
                onClick={[Function]}
              >
                MYR
              </p>
              <p
                onClick={[Function]}
              >
                MZN
              </p>
              <p
                onClick={[Function]}
              >
                NAD
              </p>
              <p
                onClick={[Function]}
              >
                NGN
              </p>
              <p
                onClick={[Function]}
              >
                NIO
              </p>
              <p
                onClick={[Function]}
              >
                NOK
              </p>
              <p
                onClick={[Function]}
              >
                NPR
              </p>
              <p
                onClick={[Function]}
              >
                NZD
              </p>
              <p
                onClick={[Function]}
              >
                OMR
              </p>
              <p
                onClick={[Function]}
              >
                PAB
              </p>
              <p
                onClick={[Function]}
              >
                PEN
              </p>
              <p
                onClick={[Function]}
              >
                PGK
              </p>
              <p
                onClick={[Function]}
              >
                PHP
              </p>
              <p
                onClick={[Function]}
              >
                PKR
              </p>
              <p
                onClick={[Function]}
              >
                PLN
              </p>
              <p
                onClick={[Function]}
              >
                PYG
              </p>
              <p
                onClick={[Function]}
              >
                QAR
              </p>
              <p
                onClick={[Function]}
              >
                RON
              </p>
              <p
                onClick={[Function]}
              >
                RSD
              </p>
              <p
                onClick={[Function]}
              >
                RUB
              </p>
              <p
                onClick={[Function]}
              >
                RWF
              </p>
              <p
                onClick={[Function]}
              >
                SAR
              </p>
              <p
                onClick={[Function]}
              >
                SBD
              </p>
              <p
                onClick={[Function]}
              >
                SCR
              </p>
              <p
                onClick={[Function]}
              >
                SDG
              </p>
              <p
                onClick={[Function]}
              >
                SEK
              </p>
              <p
                onClick={[Function]}
              >
                SGD
              </p>
              <p
                onClick={[Function]}
              >
                SHP
              </p>
              <p
                onClick={[Function]}
              >
                SLE
              </p>
              <p
                onClick={[Function]}
              >
                SLL
              </p>
              <p
                onClick={[Function]}
              >
                SOS
              </p>
              <p
                onClick={[Function]}
              >
                SRD
              </p>
              <p
                onClick={[Function]}
              >
                STD
              </p>
              <p
                onClick={[Function]}
              >
                SVC
              </p>
              <p
                onClick={[Function]}
              >
                SYP
              </p>
              <p
                onClick={[Function]}
              >
                SZL
              </p>
              <p
                onClick={[Function]}
              >
                THB
              </p>
              <p
                onClick={[Function]}
              >
                TJS
              </p>
              <p
                onClick={[Function]}
              >
                TMT
              </p>
              <p
                onClick={[Function]}
              >
                TND
              </p>
              <p
                onClick={[Function]}
              >
                TOP
              </p>
              <p
                onClick={[Function]}
              >
                TRY
              </p>
              <p
                onClick={[Function]}
              >
                TTD
              </p>
              <p
                onClick={[Function]}
              >
                TWD
              </p>
              <p
                onClick={[Function]}
              >
                TZS
              </p>
              <p
                onClick={[Function]}
              >
                UAH
              </p>
              <p
                onClick={[Function]}
              >
                UGX
              </p>
              <p
                onClick={[Function]}
              >
                USD
              </p>
              <p
                onClick={[Function]}
              >
                UYU
              </p>
              <p
                onClick={[Function]}
              >
                UZS
              </p>
              <p
                onClick={[Function]}
              >
                VEF
              </p>
              <p
                onClick={[Function]}
              >
                VES
              </p>
              <p
                onClick={[Function]}
              >
                VND
              </p>
              <p
                onClick={[Function]}
              >
                VUV
              </p>
              <p
                onClick={[Function]}
              >
                WST
              </p>
              <p
                onClick={[Function]}
              >
                XAF
              </p>
              <p
                onClick={[Function]}
              >
                XAG
              </p>
              <p
                onClick={[Function]}
              >
                XAU
              </p>
              <p
                onClick={[Function]}
              >
                XCD
              </p>
              <p
                onClick={[Function]}
              >
                XDR
              </p>
              <p
                onClick={[Function]}
              >
                XOF
              </p>
              <p
                onClick={[Function]}
              >
                XPF
              </p>
              <p
                onClick={[Function]}
              >
                YER
              </p>
              <p
                onClick={[Function]}
              >
                ZAR
              </p>
              <p
                onClick={[Function]}
              >
                ZMK
              </p>
              <p
                onClick={[Function]}
              >
                ZMW
              </p>
              <p
                onClick={[Function]}
              >
                ZWL
              </p>
            </div>
          </div>
          <input
            className="_Input_aa56d9"
            disabled={true}
            name="rate"
            placeholder="Rate"
            type="number"
            value={0}
          />
        </div>
      </div>
      <div
        className="_Form_Column_aa56d9"
      >
        <div
          className="display-center"
        >
          <button
            className="button background-teal c-white"
          >
            Generate
          </button>
        </div>
        <div
          className="_Form_Column_aa56d9"
        >
          <h3
            className="mr-4 ml-4"
          >
            Result
          </h3>
          <input
            className="_Input_aa56d9"
            disabled={true}
            placeholder="Rate"
            value=" $ 0"
          />
        </div>
      </div>
    </form>
  </div>
</div>
`;
